api/hearings/hearing.mock.ts

```diff
@@ -2,16 +2,20 @@ import { AxiosRequestConfig } from 'axios';
 import MockAdapter from 'axios-mock-adapter';
 import { HttpMockAdapter } from '../common/httpMockAdapter';
 import { CASE_FLAG_REFERENCE_VALUES } from './data/caseFlagReference.mock.data';
+import { generateHearingActuals } from './data/hearing-actuals.mock.data';
+import { HEARING_RESPONSE_RESULT } from './data/hearingResponse.mock.data';
 import { EMPTY_HEARINGS_LIST, HEARINGS_LIST } from './data/hearings.mock.data';
+import { getRandomElement } from './data/mock-data-utils';
 import { ALL_REF_DATA } from './data/reference.mock.data';
 import { SERVICE_HEARING_VALUES } from './data/serviceHearingValues.mock.data';
-import { HEARING_RESPONSE_RESULT } from './data/hearingResponse.mock.data';
 
 export const init = () => {
   const mock: MockAdapter = HttpMockAdapter.getInstance();
 
   const getHearingsUrl = /https:\/\/hearings.aat.service.core-compute-aat.internal\/hearings\/[0-9]{16}/;
 
+  const getHearingsActualsUrl = /https:\/\/hearings.aat.service.core-compute-aat.internal\/actuals\/[0-9]{16}/;
```

Comment:
hearing id is not 16 digits, and we don't know what it looks like yet, so could we use [\w]* as the getHearingInfoUrl below?